app/helpers/i18n_helper.rb

```diff
@@ -12,12 +12,16 @@ module I18nHelper
   #  - {parent_controller}.global.{key}
   #  - ...
   #  - global.{key}
-  def translate_inheritable(key, variables = {}) # rubocop:disable Metrics/CyclomaticComplexity
+  def translate_inheritable(key, variables = {}) # rubocop:disable Metrics/CyclomaticComplexity, Metrics/AbcSize, Metrics/PerceivedComplexity
     partial = defined?(@virtual_path) ? @virtual_path.gsub(/.*\/_?/, '') : nil
     defaults = inheritable_translation_defaults(key, partial)
     variables[:default] ||= defaults
     variables[:model] ||= model_class&.model_name&.human if respond_to?(:model_class)
-    t(defaults.shift, **variables).upcase_first
+    if !variables.key?(:upcase) || variables[:upcase]
```

Comment:
Instead of introducing an if else here causing Rubocop to complain about AbcSize and complexity I implore you to try this in one line or split it up into smaller methods. 

Rubocop is here for a reason :wink: